### ISSUE TITLE
False positive when using a `do` scope

### ISSUE THREAD

**Issue by Page-** [za-creature/coffeescope#22]
``` coffee
for v, k in { foo: 'bar' }
    do (v, k) ->
        console.log(v, k)
```

gives me

```
     ⚡ #1: Variable "v" is never used.
     ⚡ #1: Variable "k" is never used.
     ⚡ #2: Variable "v" is never used (first defined on line 1).
     ⚡ #2: Variable "k" is never used (first defined on line 1).
```

**Comment by za-creature** [comment]
Thanks for the report. Fixed and released in 0.3.8

Sorry for the delay, I had a bit of a busy period

**Comment by Page-** [comment]
No worries, thanks for fixing this! :)
